### DISCUSSION TITLE
SOLVED: fuzzy sound from IQaudio DAC Pro

### DISCUSSION THREAD

**Discussion by barriousc** [rern/rAudio#962] [Fixed Issues]
I am having an issue with the DAC where the sound gets "fuzzy" on higher pitched vocals or instruments. 

I have the Pi, a CD player, and a turntable all plugged into the same receiver and only the Pi has this "fuzzy" sound problem. 

I verified that the problem is, indeed, with the DAC by plugging directly into the Pi's headphone jack and the fuzzy sound disappeared.

Specs/settings of my system are below.

Is there something I am missing?

Thanks in advance...
_______________________________________
### System:
**System:**
Version: rAudio 1 • 20220514
Kernel: 5.15.35-1 • aarch64
Hardware: Raspberry Pi 3B Rev 1.2
SoC: BCM2837 • 952MiB
CPU: 4 x Cortex-A53 @ 1.2 GHz

**GPIO Devices:**
Audio - I's: IQAudio DAC Pro

### Player:
**Output:**
Device: IQaudiODAC
Mixer Device: Digital
Volume Control: None/0dB

**Bit - Perfect:**
No Volume: enabled

**Comment by R101G** [comment]
@barriousc 
Assuming you have checked for low voltage warnings etc, this could be due to RF interference. If you are using wifi, turn it off and try a cable connection instead. Also see if grounding the connecting leads at the Pi end makes any difference.

**Comment by barriousc** [comment]
Ok, I am a dummy.

I do not have the iQaudiO DAC Pro.

I have the iQaudiO Pi-DAC PRO.

Problem solved.
